src/main/scala/com/github/unchama/seichiassist/util/BreakUtil.scala

```diff
@@ -147,26 +147,32 @@ object BreakUtil {
     }
   }
 
-  def isProtectedNetherQuartzBlock(player: Player, checkTarget: Block): Boolean = {
-    checkTarget.getType match {
-      // 鉱石ブロックの方はプロテクトの対象外
-      case Material.QUARTZ_BLOCK | Material.QUARTZ_STAIRS =>
-        if (
-          !SeichiAssist
-            .instance
-            .breakSkillTargetConfigSystem
-            .api
-            .breakSkillTargetConfig(player, BreakSkillTargetConfigKey.MadeFromNetherQuartz)
-            .unsafeRunSync()
-        ) {
-          ActionBarMessageEffect(s"${RED}スキルでのネザー水晶類ブロックの破壊は無効化されています")
-            .run(player)
-            .unsafeRunSync()
-          true
-        } else {
-          false
-        }
-      case _ => false
+  /**
+   * ブロックがネザー水晶類破壊フラグの保護対象かどうか判定する。
+   * ここでいう「ネザー水晶類」に鉱石は含まれない。
+   * @param player プレイヤー
+   * @param checkTarget 判定するブロック
+   * @return ブロックがネザー水晶類破壊フラグの保護対象かどうか
```

Comment:
```suggestion
   * @return `checkTarget`が保護対象のネザー水晶類ブロックかどうか
```

`フラグ`とはどこでも呼んでいないので、ちょっと意味が違うと思いました。